### ISSUE TITLE
NPE while deploy vm on a IPv6-only shared network

### ISSUE THREAD

**Issue by weizhouapache** [apache/cloudstack#5640]
Got a NPE
```
java.lang.NullPointerException
        at com.cloud.network.Networks$BroadcastDomainType.getSchemeValue(Networks.java:177)
        at com.cloud.network.Networks$BroadcastDomainType.getValue(Networks.java:231)
        at com.cloud.hypervisor.kvm.resource.BridgeVifDriver.plug(BridgeVifDriver.java:209)
        at com.cloud.hypervisor.kvm.resource.LibvirtComputingResource.createVif(LibvirtComputingResource.java:3015)
        at com.cloud.hypervisor.kvm.resource.LibvirtComputingResource.createVifs(LibvirtComputingResource.java:2710)
        at com.cloud.hypervisor.kvm.resource.wrapper.LibvirtStartCommandWrapper.execute(LibvirtStartCommandWrapper.java:81)
        at com.cloud.hypervisor.kvm.resource.wrapper.LibvirtStartCommandWrapper.execute(LibvirtStartCommandWrapper.java:46)
        at com.cloud.hypervisor.kvm.resource.wrapper.LibvirtRequestWrapper.execute(LibvirtRequestWrapper.java:78)
        at com.cloud.hypervisor.kvm.resource.LibvirtComputingResource.executeRequest(LibvirtComputingResource.java:1768)
```

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
management server
~~~

##### CLOUDSTACK VERSION

~~~
4.16.0.0
~~~

##### CONFIGURATION

advanced zone

##### OS / ENVIRONMENT

Mgmt server: Ubuntu 20.04
Hypervisor: Ubuntu 20.04

##### STEPS TO REPRODUCE

~~~
1. create a shared network with only ipv6
2. deploy a vm
~~~

##### EXPECTED RESULTS

~~~
succeed
~~~

##### ACTUAL RESULTS

~~~
Got exception, see the description
~~~

**Comment by rhtyd** [comment]
Partially fixed in https://github.com/apache/cloudstack/pull/5648

**Comment by sureshanaparti** [comment]
Fixed in #5810
